### ISSUE TITLE
Incomplete records may accumulate

### ISSUE THREAD

**Issue by gmcressman** [ICTatRTI/coconut-mobile#52]
After selecting Complete, app generates a new incomplete record. These can accumulate.

**Comment by mikeymckay** [comment]
Yeah - normally the new incomplte record isn't saved until some new data is added, but because we pre-fill the ID and pre-set the hospital transfer, changes are recorded and the doc is saved. I expect that these will just be ignored. If anybody needs to find an incomplete record, it will be there and can probably be found using the search.
